server/android/src/android.rs

```diff
@@ -23,6 +23,8 @@ pub mod android {
     static SERVER_BUCKET: Mutex<Option<ServerBucket>> = Mutex::new(None);
 
     #[no_mangle]
+    /// # Safety
+    /// TODO: Add why this is unsafe
```

Comment:
Good question, looks like its not needed? have you tried without it? I copied it from an example, maybe it was needed at some earlier version of the jni...